dags/burnham.py

```diff
@@ -143,7 +143,7 @@
   WHERE
     metrics.string.test_name = "{DEFAULT_TEST_NAME}"
   LIMIT
-    20 ),
+    40 ),
```

Comment:
@hackebrot - Do you remember if there is a specific reason for the LIMIT clauses used here? If it's only there to reduce the potential for accidentally returning a large result, I think the likelihood of that causing issues is low enough that we can remove the limit clauses.

Reply:
I believe we've only added the `LIMIT` clauses because it's best practice and we knew that we wouldn't be seeing more pings for a given test run. I'd be happy to increase that to something higher or if that's your recommendation remove that entirely.

Reply:
I'd be happy for us to remove it entirely. The danger of lost time due to debugging unexpected behavior seems bigger than any query cost or efficiency gain here.